src/main/java/com/extrahardmode/config/RootNode.java

```diff
@@ -53,6 +53,22 @@
      */
     PRINT_COMMENTS("Print Comments", VarType.BOOLEAN, true, "If comments like this should be printed"),
     /**
+    * Globally disables certain modules. These settings might improve performance if these modules is not used. 
```

Comment:
That's not a bad idea. I cant think of that many exist variables to use tho. I guess you can check if that torch height is zero, but I dont know how to make that support multi-world configs.